http/server.go

```diff
@@ -23,7 +23,7 @@ func (a *AppServer) initialize() {
 
 	mux := http.NewServeMux()
 	mux.Handle("/api/v1/", privateChain)
-	mux.Handle("/api/", public)
+	mux.Handle("/api/", Logger(public))
```

Comment:
I think we should have a logger for /api/v1/ as well for now

Reply:
[We have](https://github.com/eduboard/backend/pull/30/files#diff-f13600ca159ef5d2d6395955b8b61c4aR22) 😊